java-manta-it/src/test/java/com/joyent/manta/client/MantaClientAuthenticationIT.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2013-2019, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.client;
+
+import com.joyent.manta.config.*;
+import com.joyent.manta.exception.ConfigurationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.io.IOException;
+import java.util.UUID;
+
+/**
+ * Tests client authentication, sub-user functionality and other issues
+ * associated with it for the {@link MantaClient} class.
+ *
+ * @author <a href="https://github.com/nairashwin952013">Ashwin A Nair</a>-
+ */
+public class MantaClientAuthenticationIT {
```

Comment:
Question: Is this file correctly referenced in the testng.xml file so that it is executed during Maven runs?

Reply:
It is already included implicitly within the ```testng-it.xml``` since it comes under the package ```com.joyent.manta.client.*``` for the test-name:```Manta Client Integration Tests [Unencrypted]```